website/docs/docs/dbt-versions/release-notes/72-Feb-2024/feb-sl-updates.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "dbt Semantic Layer updates and fixes for February 2024"
+description: "Feb 2024 updates for dbt Semantic Layer & MetricFlow: New export features, enhanced date filters, Tableau, Google Sheets & GraphQL API enhancements, and bug fixes."
+sidebar_label: "Updates and fixes: dbt Semantic Layer"
+sidebar_position: 05
+tags: [Feb-2024]
+date: 2024-02-29
+---
+
+The dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability for the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and [MetricFlow](/docs/build/about-metricflow).
+
+The following list explains the new features, updates, and fixes for February 2024 in more detail.
+
+## dbt Semantic Layer
+**New features**
+
+- **New export capabilities** &mdash; You can access a list of your [exports](/docs/use-dbt-semantic-layer/exports) with the new list saved-queries command by adding `--show-exports`
+- **Relative date filters in Tableau** &mdash; The dbt Semantic Layer and [Tableau Connector](/docs/use-dbt-semantic-layer/tableau) now supports relative date filters in Tableau.
+
+**Updates**
+
+- **Google Sheets integration** &mdash; The dbt Semantic Layer [Google Sheets integration](/docs/use-dbt-semantic-layer/gsheets) now exposes a note on the cell where the data was requested, indicating clearer data requests. The integration also now exposes a new **Time Range** options, which allows you to quickly select date ranges.
+- **GraphQL API enhancements** &mdash; The [GraphQL API](/docs/dbt-cloud-apis/sl-graphql) includes a `requiresMetricTime` parameter to better handle metrics that must be grouped by time. (Certain metrics defined in MetricFlow can't be looked at without a time dimension).
+
+**Bug fixes**
+
+- **Date filter with Tableau Connector** &mdash; The Tableau Connector returns a date filter when filtering by dates. Previously it was erroneously returning a timestamp filter.
+
+## MetricFlow
+**New features**
+
+- **Use exports to materialize saved queries** &mdash; [Exports](/docs/use-dbt-semantic-layer/exports#define-exports) allow you to materialize a saved query as a table or view in your data platform. By using exports, you can unify metric definitions in your data platform and query them as you would any other table or view.
```

Comment:
isn't exports a SL feature? if so, shouldn't it belong in the SL area or is it MF?

Reply:
agreed @Jstein77 what do you think?